### PR TITLE
Fix incorrect argparse descriptions on examples

### PR DESCRIPTION
**Pull request by guilhermedemouraa**

### CODE REVIEW

File: py/examples/pendant_state/main.py [hunk 1]
```diff
@@ -45,7 +45,9 @@ async def main(service_config_path: Path) -> None:
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(prog="Stream PendantState messages the canbus service.")
+    parser = argparse.ArgumentParser(
+        prog="python main.py", description="Stream PendantState messages the canbus service."
```
Comment (Hackerman342) [inline py/examples/pendant_state/main.py:49]:
```suggestion
        prog="python main.py", description="Stream PendantState messages from the canbus service."
```

File: py/examples/service_counter/client.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ async def command_reset(client: CounterClient) -> None:
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(prog="farm-ng-service")
+    parser = argparse.ArgumentParser(prog="python client.py", description="Farm-ng service example")
```
Comment (Hackerman342) [inline py/examples/service_counter/client.py:57]: This one is a client to the counter service

File: py/examples/service_counter/counter.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ async def serve(self) -> None:
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(prog="farm-ng-service")
+    parser = argparse.ArgumentParser(prog="python counter.py", description="Farm-ng service example")
```
Comment (Hackerman342) [inline py/examples/service_counter/counter.py:63]:
```suggestion
    parser = argparse.ArgumentParser(prog="python counter.py", description="Farm-ng counter service example")
```

File: py/examples/service_propagation/supervisor.py [hunk 4]
```diff
@@ -71,7 +71,7 @@ async def serve(self) -> None:
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(prog="farm-ng-service")
+    parser = argparse.ArgumentParser(prog="python supervisor.py", description="Farm-ng service propagation example")
```
Comment (Hackerman342) [inline py/examples/service_propagation/supervisor.py:74]:
```suggestion
    parser = argparse.ArgumentParser(prog="python supervisor.py", description="Farm-ng service propagation example supervisor")
```

File: py/examples/service_propagation/storage.py [hunk 5]
```diff
@@ -83,7 +83,7 @@ async def serve(self) -> None:
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(prog="farm-ng-service")
+    parser = argparse.ArgumentParser(prog="python storage.py", description="Farm-ng service propagation example")
```
Comment (Hackerman342) [inline py/examples/service_propagation/storage.py:86]:
```suggestion
    parser = argparse.ArgumentParser(prog="python storage.py", description="Farm-ng service propagation example storage service")
```

File: py/examples/service_propagation/agent.py [hunk 6]
```diff
@@ -80,7 +80,7 @@ async def serve(self) -> None:
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(prog="farm-ng-service")
+    parser = argparse.ArgumentParser(prog="python agent.py", description="Farm-ng service propagation example")
```
Comment (Hackerman342) [inline py/examples/service_propagation/agent.py:83]:
```suggestion
    parser = argparse.ArgumentParser(prog="python agent.py", description="Farm-ng service propagation example agent")
```

**Review by Hackerman342 — APPROVED** [review]
Amazing! Thank you

File: py/examples/pendant_state/main.py (hunk 1 above)
Comment (guilhermedemouraa) [reply py/examples/pendant_state/main.py]: Good catch! Addressed here: 4e1e16e0cc15ed2c832c3359927e54433a5e355a

File: py/examples/service_counter/client.py (hunk 2 above)
Comment (guilhermedemouraa) [reply py/examples/service_counter/client.py]: Clearer description added here: 4e1e16e0cc15ed2c832c3359927e54433a5e355a

File: py/examples/service_propagation/supervisor.py (hunk 4 above)
Comment (guilhermedemouraa) [reply py/examples/service_propagation/supervisor.py]: Clearer description added here: https://github.com/farm-ng/farm-ng-amiga/commit/4e1e16e0cc15ed2c832c3359927e54433a5e355a

File: py/examples/service_propagation/storage.py (hunk 5 above)
Comment (guilhermedemouraa) [reply py/examples/service_propagation/storage.py]: Clearer description added here: https://github.com/farm-ng/farm-ng-amiga/commit/4e1e16e0cc15ed2c832c3359927e54433a5e355a

File: py/examples/service_propagation/agent.py (hunk 6 above)
Comment (guilhermedemouraa) [reply py/examples/service_propagation/agent.py]: Clearer description added here: https://github.com/farm-ng/farm-ng-amiga/commit/4e1e16e0cc15ed2c832c3359927e54433a5e355a

**Pull request merged**